.snippets/code/tutorials/polkadot-sdk/system-chains/asset-hub/papi-console-output.html

```diff
@@ -0,0 +1,4 @@
+<div id="termynal" data-termynal>
```

Comment:
You should include the command and the output. So I would add the command and don't forget to add the file path. There should be other examples of this in the code base.